server/api/urls.py

```
@@ -1,10 +1,11 @@
 from django.urls import path
-from .views import UserRegistrationView, SessionView, CurrentUserView, SuperView, UserIDView, SuperIDView, PostView, UserUNameGet,LikesUNameGet,PostsUNameGet,SupersUNameGet
+from .views import *
```

Ruff (0.8.2)

`from .views import *` used; unable to detect undefined names

(F403)